bin/server_start.sh

```diff
@@ -73,7 +73,8 @@ fi
 # This needs to be exported for standalone mode so drivers can connect to the Spark cluster
 export SPARK_HOME
 
-$SPARK_HOME/bin/spark-submit --class $MAIN --driver-memory 5G \
+# DSE_BIN is set in settings.sh
+$DSE_HOME/bin/dse spark-submit --class $MAIN --driver-memory 5G \
```

Comment:
I understand that this is a third-party script. However IMHO it lacks of few fixes: try `shellcheck`

```
In server_start.sh line 9:
  cd $(dirname $0)
     ^-- SC2046: Quote this to prevent word splitting.
               ^-- SC2086: Double quote to prevent globbing and word splitting.


In server_start.sh line 29:
conffile=$(ls -1 $appdir/*.conf | head -1)
           ^-- SC2012: Use find instead of ls to better handle non-alphanumeric filenames.
                 ^-- SC2086: Double quote to prevent globbing and word splitting.


In server_start.sh line 36:
  . $appdir/settings.sh
    ^-- SC2086: Double quote to prevent globbing and word splitting.


In server_start.sh line 49:
if [ -f "$pidFilePath" ] && kill -0 $(cat "$pidFilePath"); then
                                    ^-- SC2046: Quote this to prevent word splitting.


In server_start.sh line 77:
$DSE_HOME/bin/dse spark-submit --class $MAIN --driver-memory 5G \
^-- SC2086: Double quote to prevent globbing and word splitting.


In server_start.sh line 80:
  $@ $appdir/spark-job-server.jar $conffile 2>&1 &
  ^-- SC2068: Add double quotes around ${@}, otherwise it's just like $* and breaks on spaces.
     ^-- SC2086: Double quote to prevent globbing and word splitting.
                                  ^-- SC2086: Double quote to prevent globbing and word splitting.


In server_start.sh line 81:
echo $! > $pidFilePath
     ^-- SC2086: Double quote to prevent globbing and word splitting.
          ^-- SC2086: Double quote to prevent globbing and word splitting.
```